engine/engine/content/builtins/scripts/mobdebug.lua

```diff
@@ -592,16 +592,21 @@ local function get_server_data(file)
     return
   end
   checkcount = mobdebug.checkcount
-  if coroutine.status(coro_debugger) == "suspended" then
-    local status, res = cororesume(coro_debugger, nil,nil,file)
+  if corostatus(coro_debugger) == "suspended" then
+    local state_vars = capture_vars(3)
+    local status, res = cororesume(coro_debugger, nil, state_vars, file)
```

Comment:
Current state should be taken to set it as environment to be able to Evaluate Lua code
But default `vars` should be overwritten to be able to restore environment of the debugger itself.

Reply:
What's the difference between corostatus and coroutine.status?

Reply:
It's just a mobdebug wrapper for coroutine.status(): https://github.com/defold/defold/blob/de77e1c99f36a56e64594bc93d6098580dc471ee/engine/engine/content/builtins/scripts/mobdebug.lua#L65